diff --git a/CMakeLists.txt b/CMakeLists.txt
index b080e50..70be054 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -25,17 +25,9 @@ endif()
  
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
  
-# user defined options for MKL
-option(MKL_USE_parallel "Use MKL parallel" False)
-option(MKL_USE_sdl "Single Dynamic Library or static/dynamic" False)
-set(BLA_STATIC True CACHE STRING "Use static linking instead of dynamic linking for MKL")
-set(MKL_USE_interface "lp64" CACHE STRING "for Intel(R)64 compatible arch: ilp64/lp64 or for ia32 arch: cdecl/stdcall")
-
-
 set(BOOST_ALL_NO_LIB 1)
 find_package(Boost 1.60.0)
 find_package(Eigen3 3.3.7 NO_MODULE)
-find_package(MKL) 
 
 include_directories(include)
 
@@ -103,12 +95,6 @@ else()
     )
 endif()
 
-if (MKL_FOUND)
-	target_compile_definitions(xgandalf PUBLIC ${MKL_DEFINITIONS} USE_MKL EIGEN_USE_MKL_ALL)
-	target_link_libraries(xgandalf PRIVATE ${MKL_LIBRARIES})
-	target_include_directories(xgandalf PRIVATE ${MKL_INCLUDE_DIR})
-endif(MKL_FOUND)
-
 # Test whether the compiler is Microsoft Visual C(++).
 if(CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
   target_compile_options(xgandalf PUBLIC /W2 /wd4305 /wd4244 /wd4099)
@@ -152,4 +138,4 @@ install(
 # xgandalf.pc
 configure_file(xgandalf.pc.in xgandalf.pc)
 install(FILES ${CMAKE_BINARY_DIR}/xgandalf.pc
-        DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
\ No newline at end of file
+        DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
